README.md

```diff
@@ -188,10 +188,10 @@ the `discovery-server-ws/src` directory:
 
 |            PACKAGE                 |                              URL                        |    BRANCH   |
 |:-----------------------------------|:--------------------------------------------------------|:-----------:|
-| eProsima/Fast-CDR:                 | https://github.com/eProsima/Fast-CDR.git                |    master   |
-| eProsima/Fast-DDS:                | https://github.com/eProsima/Fast-DDS.git               |    master   |
-| eProsima/Discovery-Server:         | https://github.com/eProsima/Discovery-Server.git        |    master   |
-| eProsima/foonathan_memory_vendor:  | https://github.com/eProsima/foonathan_memory_vendor.git |    master   |
+| fastcdr:                 | https://github.com/eProsima/Fast-CDR.git                |    master   |
+| fastdds:                 | https://github.com/eProsima/Fast-DDS.git               |    master   |
+| discovery-server:         | https://github.com/eProsima/Discovery-Server.git        |    master   |
+| foonathan_memory_vendor:  | https://github.com/eProsima/foonathan_memory_vendor.git |    master   |
 | leethomason/tinyxml2:              | https://github.com/leethomason/tinyxml2.git             |    master   |
```

Comment:
Though it is not really necessary, previously, the table was aligned for aesthetic purposes. I do not mind to avoid introducing blank spaces, though then I suggest removing the unnecessary blank spaces as well. Else, please keep the table aligned